clients/middleware/authorization.go

```diff
@@ -98,6 +110,15 @@ func (am *authorizationMiddleware) CreateClients(ctx context.Context, session au
 		return []clients.Client{}, []roles.RoleProvision{}, errors.Wrap(err, errDomainCreateClients)
 	}
 
+	params := map[string]interface{}{
+		"quantity": len(client),
+		"entity":   client,
+	}
```

Comment:
```suggestion
		"entities": []client.Client{...client},
```
Becayse client is variadic `...client.Clinet`